Microsoft.Toolkit.Uwp.UI.Controls/RangeSelector/RangeSelector.xaml

```diff
@@ -8,7 +8,7 @@
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="controls:RangeSelector">
-                    <Grid Height="24">
+                    <Grid Height="40">
```

Comment:
Hello unfortunately this is a breaking change.
People who developed applications using this control don't expect this to change suddenly.